huxley/core/migrations/0001_initial.py

```diff
@@ -252,8 +252,8 @@ def backwards(self, orm):
             'state': ('django.db.models.fields.CharField', [], {'max_length': '16'}),
             'times_attended': ('django.db.models.fields.PositiveSmallIntegerField', [], {'default': '0'}),
             'waitlist': ('django.db.models.fields.BooleanField', [], {'default': 'False'}),
-            'zip_code': ('django.db.models.fields.CharField', [], {'max_length': '16'})
+            'zip_code': ('django.db.models.fields.CharField', [], {'max_length': '16'}),
```

Comment:
Looks like the assignments_finalized attribute hasn't been migrated so the Travis build is failing.